bouncer/commands/get_usdc_balance.ts

```diff
@@ -7,19 +7,12 @@
 // For example: ./commands/get_usdc_balance.ts 0xcf1dc766fc2c62bef0b67a8de666c8e67acf35f6
 // might print: 100.2
 
-import { runWithTimeout, getContractAddress } from '../shared/utils';
+import { executeWithTimeout, getContractAddress } from '../shared/utils';
 import { getErc20Balance } from '../shared/get_erc20_balance';
 
 async function getUsdcBalanceCommand(ethereumAddress: string) {
   const contractAddress = getContractAddress('Ethereum', 'Usdc');
-  const balance = await getErc20Balance('Ethereum', ethereumAddress, contractAddress);
-  console.log(balance);
-  process.exit(0);
+  console.log(await getErc20Balance('Ethereum', ethereumAddress, contractAddress));
 }
 
-const ethereumAddress = process.argv[2] ?? '0';
-
-runWithTimeout(getUsdcBalanceCommand(ethereumAddress), 5000).catch((error) => {
-  console.error(error);
-  process.exit(-1);
-});
+await executeWithTimeout(getUsdcBalanceCommand(process.argv[2] ?? '0'), 5);
```

Comment:
IMO the code was more clear with `const ethereumAddress = process.argv[2] ?? '0';` uninlined (otherwise it took me a while to understand what we are reading from command line arguments). Same in other places.

Reply:
agreed

Reply:
I was suggesting that we uninline all other instances where process.argv was inlined by this PR, not just this particular line @kylezs 🙂